### PR TITLE
[i18nIgnore] [studio-DB] add studio content to DB guide

### PR DESCRIPTION
**Pull request by sarah11918**
#### Description (required)

In prep for sidebar reorg, this page removes the Astro Studio dedicated page and moves the info on to the Astro DB guide page.

No content/text changes **except heading levels** of the newly added content and link updates.

- [x] add page redirect from `/[lang]/recipes/studio/` to /[lang]/guides/astro-db/#astro-studio` in Vercel file
- [x] remove Astro Studio from sidebar nav under Recipes
- [ ] update `isSubpage` logic? (can wait until sidebar reorg PR itself)
- [x] waiting for translators to remove the Studio page themselves in OTHER locales so that they have the text to copy and paste!!
- [x] `zh-cn`
- [x] `fr`
- [x] `ar`
- [x] `de`
- [x] `es`
- [x] `it`
- [x] `hi`
- [x] `ja`
- [x] `ko`
- [x] `pl`
- [x] `pt-br`
- [x] `ru`
- [x] `zh-tw`

### CODE REVIEW

File: src/content/docs/en/guides/integrations-guide/db.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import ReadMore from '~/components/ReadMore.astro';
 import Badge from '~/components/Badge.astro';
 
 
-Astro DB is a fully-managed SQL database designed for the Astro ecosystem: develop locally in Astro and deploy from your [Astro Studio dashboard](/en/recipes/studio/).
+Astro DB is a fully-managed SQL database designed for the Astro ecosystem: develop locally in Astro and deploy from your [Astro Studio dashboard](#astro-studio).
```
Comment (liruifengv) [inline src/content/docs/en/guides/integrations-guide/db.mdx:15]:
```suggestion
Astro DB is a fully-managed SQL database designed for the Astro ecosystem: develop locally in Astro and deploy from your [Astro Studio dashboard](/en/guides/astro-db/#astro-studio).
```

fix this link

File: src/content/docs/en/guides/integrations-guide/db.mdx [hunk 2]
```diff
@@ -196,7 +196,7 @@ Each foreign key configuration object accepts the following properties:
 
 ## Astro DB CLI reference
 
-Astro DB includes a set of CLI commands to interact with your hosted project database and your [Astro Studio](/en/recipes/studio/) account. 
+Astro DB includes a set of CLI commands to interact with your hosted project database and your [Astro Studio](#astro-studio) account.
```
Comment (liruifengv) [inline src/content/docs/en/guides/integrations-guide/db.mdx:199]:
```suggestion
Astro DB includes a set of CLI commands to interact with your hosted project database and your [Astro Studio](/en/guides/astro-db/#astro-studio) account. 
```

**Review by liruifengv — COMMENTED** [review]
Fix links

**Comment by jsparkdev** [comment]
Korean version of PR: #8446

**Comment by sarah11918** [comment]
Noticing that the original `es` Studio recipe translation was missing the first section "create a new studio project".

So, `/es/guides/astro-db/` will still need updating!

**Review by yanthomasdev — APPROVED** [review]

**Pull request merged**